docs/development/cluster.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+name: compliantkubernetes
```

Comment:
I think they recommend not specifying the name in the manifest (also you can't delete a cluster just specifying the manifest)
```suggestion
```